src/overlays/actors/ovl_En_Sob1/z_en_sob1.c

```diff
@@ -1663,7 +1668,7 @@ void EnSob1_BombShopkeeper_Draw(Actor* thisx, GlobalContext* globalCtx) {
 
     OPEN_DISPS(globalCtx->state.gfxCtx);
     func_8012C28C(globalCtx->state.gfxCtx);
-    gSPSegment(POLY_OPA_DISP++, 0x08, Lib_SegmentedToVirtual(object_rs_Tex_005458));
+    gSPSegment(POLY_OPA_DISP++, 0x08, Lib_SegmentedToVirtual(gBombShopkeeperEyesTex));
```

Comment:
Presumably this is both eyes in one texture?

Reply:
Oh that should be EyeTex. Its just a single eye.

Reply:
Note I also changed some of the zora stuff to be singular as well since it makes more sense.